pact/provider/core/src/test/java/org/arquillian/algeron/pact/provider/core/ArrayUtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package org.arquillian.algeron.pact.provider.core;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ArrayUtilsTest {
+
+    @Test
+    public void should_match_any_possible_pattern() {
+        String[] states = new String[]{"I have (\\d+) cukes in my belly", "The following animals: (.*)"};
+        assertThat(ArrayUtils.matches(states, "I have 36 cukes in my belly")).isEqualTo(true);
```

Comment:
Is `ArrayUtils` really the best name for functionalities it has to offer? What I'm afraid of, like it happens with most of the `*Utils`, that at some point it will become one big bag of any possible `Array` operation (if not more!) we will be in need for.